src-electron/generator/helper-zcl.js

```diff
@@ -730,33 +730,38 @@ function zcl_attributes_client(options) {
  * Iterator over the server attributes. If it is used at toplevel, if iterates over all the server attributes
  * in the database. If used within zcl_cluster context, it iterates over all the server attributes
  * that belong to that cluster.
- *
+ * Available Options:
+ * - removeKey: Removes one of the keys from the map(for eg keys in db-mapping.js)
```

Comment:
Again, why only one key, if this is useful at all?

Reply:
Explained above